python/ray/tune/impl/tuner_internal.py

```diff
@@ -87,10 +95,14 @@ def __init__(
         self._tune_config = tune_config or TuneConfig()
         self._run_config = run_config or RunConfig()
 
+        self._missing_trainable_parameters = None
```

Comment:
Can we make it clear that those are just parameter names? Like `missing_trainable_parameter_names`